src/main/java/com/team4/testingsystem/services/impl/TestGeneratingServiceImpl.java

```diff
@@ -0,0 +1,60 @@
+package com.team4.testingsystem.services.impl;
+
+import com.team4.testingsystem.entities.ContentFile;
+import com.team4.testingsystem.entities.Question;
+import com.team4.testingsystem.entities.Test;
+import com.team4.testingsystem.enums.Modules;
+import com.team4.testingsystem.repositories.QuestionRepository;
+import com.team4.testingsystem.services.TestGeneratingService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+
+import java.util.Collection;
+
+@Service
+public class TestGeneratingServiceImpl implements TestGeneratingService {
+
+    private final QuestionRepository questionService;
+    private final ContentFilesServiceImpl contentFilesService;
+
+    @Value("${test-generating.const.question.count}")
+    private Integer count;
+
+    @Value("${test-generating.const.one-element}")
+    private Integer oneElement;
```

Comment:
One element seems to be technical value, rather the user defines constant. No need to read it from properties.